src/Icons/config/services.php

```diff
@@ -24,10 +24,15 @@
 
 return static function (ContainerConfigurator $container): void {
     $container->services()
+        ->set('cache.ux.icons')
```

Comment:
follow twig_component cache naming format https://github.com/symfony/ux/blob/2.x/src/TwigComponent/config/cache.php

Reply:
I would name this `.ux_icons.cache` to respect the best practice of prefixing service ids with the alias of the bundle for namespacing purpose.